Libs/Gtk-3.0/Classes/Expander.cs

```diff
@@ -0,0 +1,10 @@
+﻿using GObject.Native;
+
+namespace Gtk
+{
+    public partial class Expander
+    {
+        public static Expander New(string text)
+            => ObjectWrapper.WrapHandle<Expander>(Native.Expander.Instance.Methods.New(text), false);
```

Comment:
The wrapper checks if an object is already known for this pointer and afterwards calls the constructor per reflection. But as we are creating a new instance we can omit this check as we now there is no other instance and direcly call a constructor.